pkg/exchange/max/maxapi/account.go

```diff
@@ -155,7 +155,23 @@ type WithdrawState string
 
 const (
 	WithdrawStateSubmitting WithdrawState = "submitting"
+	WithdrawStateSubmitted  WithdrawState = "submitted"
 	WithdrawStateConfirmed  WithdrawState = "confirmed"
+	WithdrawStatePending    WithdrawState = "pending"
+	WithdrawStateProcessing WithdrawState = "processing"
+	WithdrawStateCanceled   WithdrawState = "canceled"
```

Comment:
Is this keyword defined by Max because it lacks an "l" and is different from another definition below?

Reply:
Yes they are max only